akka-stream/src/main/scala/akka/stream/javadsl/Flow.scala

```diff
@@ -2289,6 +2236,9 @@ final class Flow[In, Out, Mat](delegate: scaladsl.Flow[In, Out, Mat]) extends Gr
    *
    * @see [[#splitWhen]]
    */
+  @deprecated(
+    "Use .withAttributes(ActorAttributes.supervisionStrategy(equivalentDecider)) rather than a SubstreamCancelStrategy",
+    since = "2.6.19")
```

Comment:
Will need an update to 2.6.20 (or later, do that in the end when all other review feedback is 👍 )